pkg/corerp/datamodel/environment.go

```diff
@@ -38,11 +38,42 @@ func (e *Environment) ResourceTypeName() string {
 
 // EnvironmentProperties represents the properties of Environment.
 type EnvironmentProperties struct {
-	Compute    rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
-	Recipes    map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
-	Providers  Providers                                         `json:"providers,omitempty"`
-	Extensions []Extension                                       `json:"extensions,omitempty"`
-	Simulated  bool                                              `json:"simulated,omitempty"`
+	Compute      rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
+	Recipes      map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
+	Providers    Providers                                         `json:"providers,omitempty"`
+	RecipeConfig RecipeConfigProperties                            `json:"recipeConfig,omitempty"`
```

Comment:
@vishwahiremat Can we update RecipeConfig to be a pointer to RecipeConfigProperties? it can be a potentially  large object when envConfig is passed, pointer will be better.

Reply:
discussed it offline! resolving the comment